src/accessories/characteristics/targetPosition.ts

```diff
@@ -33,13 +34,8 @@ export class TargetPositionCharacteristic extends TuyaWebCharacteristic {
   }
 
   public getRemoteValue(callback: CharacteristicGetCallback): void {
-    this.accessory
-      .getDeviceState()
-      .then((data) => {
-        this.debug("[GET] %s", data?.state);
-        this.updateValue(data, callback);
-      })
-      .catch(this.accessory.handleError("GET", callback));
+    const a = <CoverAccessory>this.accessory;
+    callback && callback(null, a.target);
```

Comment:
This will fail when `this.accessory` is an instance of `WindowAccessory` which also uses this characteristic